<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoStructer</class>
 <widget class="QMainWindow" name="YoStructer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Yo Structer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(38, 37, 45);
color: rgb(215,215,215);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="directories">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>320</width>
      <height>540</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>160</height>
     </size>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="yostruct_area">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>400</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>240</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="html_area">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>400</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>240</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>608</y>
      <width>1160</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>5</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>140</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>360</width>
      <height>160</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="yostruct_area_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Область кода Yo Struct</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Область кода HTML</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Файлы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(215, 215, 215);</string>
    </property>
    <property name="text">
     <string>Консоль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transform_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Преобразовать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>570</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть в браузере</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transform_open_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>570</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Преобразовать и открыть в браузере</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="code_styles">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Стили кода</string>
    </property>
    <widget class="QRadioButton" name="pretty_style">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Аккуратный</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="oneline_style">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Однострочный</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rich_style">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Роскошный</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_6"/>
    <addaction name="action_yo_struct"/>
    <addaction name="action_html"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="help_button"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Код</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="action_yo_struct">
   <property name="text">
    <string>Сохранить .yostruct</string>
   </property>
  </action>
  <action name="action_html">
   <property name="text">
    <string>Сохранить .html</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сохранить всё</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Преобразовать</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Открыть в браузере</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Преобразовать и открыть в браузере</string>
   </property>
  </action>
  <action name="help_button">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
